src/Tribe/Customizer/Global_Elements.php

```diff
@@ -190,22 +191,71 @@ public function register_settings( WP_Customize_Section $section, WP_Customize_M
 
 		$customizer->add_setting_name( $customizer->get_setting_name( 'link_color', $section ) );
 
-		// Add an heading that is a Control only in name: it does not, actulally, control or save any setting.
+		// Add an heading that is a Control only in name: it does not, actually, control or save any setting.
 		$manager->add_control(
 			new Heading(
 				$manager,
-				$customizer->get_setting_name( 'accent_color_heading', $section ),
+				$customizer->get_setting_name( 'adjust_appearance_heading', $section ),
 				[
 					'label'   => esc_html__( 'Adjust Appearance', 'the-events-calendar' ),
 					'section' => $section->id,
 				]
 			)
 		);
 
+		$manager->add_setting(
+			$customizer->get_setting_name( 'background_color_choice', $section ),
+			[
+				'default' => $this->get_default( 'background_color_choice' ),
+			]
+		);
+
+		$manager->add_control(
+			$customizer->get_setting_name( 'background_color_choice', $section ),
+			[
+				'label'      => 'Background Color',
+				'section'    => $section->id,
```

Comment:
:warning: Array double arrow not aligned correctly; expected 5 space(s) between "'section'" and double arrow, but found 4.

<sub>hash: 4fff3509258339889905595598321c23</sub>